docs: use configuration drift when describing drift detection (#385)
hashicorp/terraform-docs-common: website/docs/cloud-docs/run/modes-and-options.mdx

-> **Version note:** Refresh-only support requires a workspace using at least Terraform CLI v0.15.4.

> [Refresh-only mode](/terraform/cli/commands/plan#planning-modes) instructs Terraform to create a plan that updates the Terraform state to match changes made to remote objects outside of Terraform. This can be useful if state drift has occurred and you would like to reconcile your state file to match the drifted remote objects. Applying a refresh-only runs will not result in further changes to remote objects.

Reviewer: Small tweaks for present tense:
```suggestion
[Refresh-only mode](/terraform/cli/commands/plan#planning-modes) instructs Terraform to create a plan that updates the Terraform state to match changes made to remote objects outside of Terraform. This is useful if state drift has occurred and you want to reconcile your state file to match the drifted remote objects. Applying a refresh-only run does not result in further changes to remote objects.
```
